cmd/experimental/kjobctl/go.mod

```diff
@@ -2,6 +2,8 @@ module sigs.k8s.io/kueue/cmd/experimental/kjobctl
 
 go 1.22.4
 
+toolchain go1.22.6
+
```

Comment:
kjobctl still requires the 1.22.4 explicitly.

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/2859/pull-kueue-test-kjobctl/1828122376194232320